docs/docs/studio.md

```diff
@@ -32,24 +41,127 @@ With Studio, it is easier to:
 
 * impersonate the user authentication headers in GraphiQL
 
-Redwood Studio is a command line tool which offers a web UI aimed at providing insights into your application via OpenTelemetry ingestion and other development conveniences like auth-impersonation within GraphiQL.
+## Features
```

Comment:
This section is helpful for providing an overview of what Studio is, but almost sounds more like marketing copy. I wonder if we could rework it to be more "docs-like." We could use this as an opportunity for education. For example: 

* we've talked about making it easy to find `n+1` issues = TBH, I'm not sure what I'm looking for. 
* Or in this case: "Observe what traces and spans indicate about your app's health." What does a healthy app look like vs a not healthy app?